README.md

```diff
@@ -49,6 +49,31 @@ async function nestedJob (child) {
   // perform some work
 })
 ```
+
+If you need to know when a queue has finished all its jobs, you can use the `drain` api.
```

Comment:
Note: "That this can be called multiple times"

Reply:
Done :)